omnibus/config/templates/datadog-agent/systemd.security.service.erb

```diff
@@ -6,7 +6,6 @@ BindsTo=datadog-agent.service
 [Service]
 Type=simple
 PIDFile=<%= install_dir %>/run/security-agent.pid
-User=dd-agent
```

Comment:
To have a consistent behavior across service managers, you need to change these other places:
- Upstart, Debian: https://github.com/DataDog/datadog-agent/blob/88b444fbff2283f33f37e10e514bcaad8de16059/omnibus/config/templates/datadog-agent/upstart_debian.security.conf.erb#L16
- Upstart, RedHat: https://github.com/DataDog/datadog-agent/blob/88b444fbff2283f33f37e10e514bcaad8de16059/omnibus/config/templates/datadog-agent/upstart_redhat.security.conf.erb#L19
- SysVInit, Debian (harmless, it's not used): https://github.com/DataDog/datadog-agent/blob/88b444fbff2283f33f37e10e514bcaad8de16059/omnibus/config/templates/datadog-agent/sysvinit_debian.security.erb#L20
- SysVInit, SUSE (harmless, it's not used): https://github.com/DataDog/datadog-agent/blob/88b444fbff2283f33f37e10e514bcaad8de16059/omnibus/config/templates/datadog-agent/sysvinit_suse.security.erb#L18

Reply:
Fixed